### DISCUSSION TITLE
API response very slow

### DISCUSSION THREAD

**Discussion by pdns90** [deepgram/community#784] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT

### Details

Hi Team,

As per our requirement, we needs output only in English language even user has given interview in any languages.

We are using https://api.deepgram.com/v1/listen?model=whisper&language=en&smart_format=true but response is very slow.

Could you please let me know if we needs to change any modal or any other changes required in coding.

Thanks,
PDN

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nikolawhallon** [comment, marked as answer]
I notice you are using `model=whisper`, whisper is a very slow model, I would recommend trying `model=nova-2` it is much faster!
